contracts/modules/capital/Pool.sol

```diff
@@ -127,17 +129,30 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
   ///
   function getPoolValueInEth() public override view returns (uint) {
 
-    uint total = address(this).balance + swapValue;
+    uint total = address(this).balance;
+
     uint assetCount = assets.length;
+    uint32 _assetsInSwapOperatorBitmap = assetsInSwapOperatorBitmap;
 
-    // Skip ETH (index 0)
-    for (uint i = 1; i < assetCount; i++) {
+    for (uint i = 0; i < assetCount; i++) {
+      Asset memory asset = assets[i];
 
-      if (assets[i].isAbandoned) {
+      if (asset.isAbandoned) {
+        continue;
+      }
+
+      address assetAddress = asset.assetAddress;
+
+      uint assetAmountInSwapOperator = isAssetInSwapOperator(i, _assetsInSwapOperatorBitmap)
+        ? assetsInSwapOperator[assetAddress]
```

Comment:
we could use the asset id as the index here instead of the address. doesn't make too big of a difference gas-wise but might be a better ux for us when trying to get values from it.